examples/stargate-prometheus-grafana/start_stargate-prometheus-grafana.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+COMPOSE_FILE=stargate-prometheus-grafana.yml
+
+# Make sure backend-1, the seed node, is up before bringing up other nodes and stargate
+docker-compose --file $COMPOSE_FILE up -d backend-1
+sleep 60
+# Bring up the 2nd C* node
+#docker-compose --file $COMPOSE_FILE up -d backend-2
+#sleep 60
+# Bring up the 3rd C* node
+#docker-compose --file $COMPOSE_FILE up -d backend-3
+#sleep 60
+# Bring up the stargate
+docker-compose --file $COMPOSE_FILE up -d stargate
+sleep 60
```

Comment:
Likewise you can do something like this to wait for stargate

```sh
echo ""
echo "Waiting for stargate to start up..."
while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost:8082/health)" != "200" ]]; do
    printf '.' 
    sleep 5
done
```

Reply:
I incorporated this wait logic.